tests/browser/encode.browser.js

```diff
@@ -32,7 +32,12 @@ test('encode.obj', function (t) {
   var circular = {}
   circular.circular = circular
   var obj3 = {bar: ['a', circular, 'c']}
-  t.equal(encode.obj(obj3, 1000), '&bar=%5B%22a%22,null,%22c%22%5D', 'Handle bad objects')
+  t.equal(encode.obj(obj3, 1000), '&bar=%5B%22a%22,%7B%7D,%22c%22%5D', 'Handle bad objects')
+
+  var circular2 = []
+  circular2.circular = circular
```

Comment:
This isn't creating the circular structure I think you're testing for. It should be like:
```suggestion
  circular2.push(circular2);
```
So that you get `circular2 = [circular2(=circular2...)]

Reply:
ty